index.js

```diff
@@ -29,3 +33,16 @@ server.use(router)
 server.listen(PORT, () => {
   console.log(`Server running on port ${PORT}`)
 })
+
+async function tooter(msg) {
```

Comment:
this can be in a separate file, e.g., `tooter.js`, ideally with a couple of tests (tests can be done later)